app/lib/theme/widgets/ZoneMapWidget.tsx

```diff
@@ -40,10 +40,28 @@ const ZoneMapWidget: React.FC<WidgetProps> = ({ formContext }) => {
     ? '/images/zone-map.png'
     : '/images/zone-map-intake-3.png';
 
+  const updatedMapIntakeFour = '/images/zone-map-intake-4.png';
+
+  const intakeNumber = formContext?.intakeNumber;
```

Comment:
Should we default to 4 to prevent broken image when intake is neither of them? 
<img width="248" alt="image" src="https://github.com/bcgov/CONN-CCBC-portal/assets/61289255/009fa7bc-8602-48bf-96da-c2df9e28a969">

Reply:
Good point, made a change now defaults for intake 4